pulseapi/entries/test_views.py

```diff
@@ -0,0 +1,275 @@
+import json
+
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+from django.test import Client
+
+from pulseapi.users.test_models import EmailUserFactory
+from pulseapi.entries.test_models import EntryFactory
+
+
+class TestEntryView(TestCase):
+    def setUp(self):
+        self.entries = [EntryFactory() for i in range(2)]
+        for entry in self.entries:
+            entry.save()
+
+        email = "test@example.org"
+        password = "password1234"
+        user = EmailUserFactory(email=email, password=password, name="test user")
+        user.save()
+
+        self.client = Client()
+        self.client.force_login(user);
+
+        values = json.loads(str(self.client.get('/nonce/').content, 'utf-8'))
+        # Set up with some curated data for all tests to use
+        postresponse = self.client.post('/entries/', data={
```

Comment:
we should probably capture this `data` object, and then for each test copy it and modify as needed.